src/index.js

```diff
@@ -2,5 +2,5 @@
 
 export {default} from './command'
 export {default as Topic} from './topic'
-export {BooleanFlag, StringFlag, NumberFlag, Flag} from './flags'
+export {boolean, string, number, Flag} from './flags'
```

Comment:
because what I want to do when I require it would be:

```js
import {flags} from 'cli-engine-command'
class Command {
  static flags = {
    app: flags.app()
  }
}
```